interface/themes/style.css

```diff
@@ -124,41 +124,17 @@ a {
 	text-decoration: none;
 }
 
-/* style taken out of the addressbook code and put here instead */
-#addressbook_list td {
-font-family: Arial, Helvetica, sans-serif;
-padding-left: 4px;
-padding-right: 4px;
-}
-#addressbook_list a, #addressbook_list a:visited, #addressbook_list a:hover {
-color:#0000cc;
-}
-#addressbook_list tr.search {
-font-size:9pt;
-font-weight:bold;
-}
-#addressbook_list tr.head {
-font-size:10pt;
-background-color:#cccccc;
-font-weight:bold;
-}
-#addressbook_list tr.detail {
-font-size:10pt;
-}
-#addressbook_list .inputtext {
-font-family:monospace;
-font-size:9pt;
-font-weight:normal;
-border-style:solid;
-border-width:1px;
-border-color: #000000;
-background-color:transparent;
-}
-#addressbook_list .button {
-font-family:sans-serif;
-font-size:8pt;
-font-weight:bold;
+/* style taken out of the address book code and put here instead
+   address book restyled to use bootstrap tables 10/23/17 sjp
+*/
+#addressbook_list {
+margin-top: 110px;
+}
+#addressbook_list .table th,td {
+font-size: 110%;
+text-align: center;
```

Comment:
I didn't know about this style.css script. Where else is it used (I would usually check myself if at home on PhpStorm, but can't check at work)?

Reply:
Nowhere I can find. Looks like someone was pulling styles from scripts and putting here. I tried hard to not use any styles other than theme but there is only so far I could go. I rather place in script but if somebody is pulling them then I don't know what to do.

Reply:
If this script is not being used anywhere, then lets remove it.
2 good options for your styling are:
1. Just place your styling in the script
2. Place your styling in all the main theme files (don't worry, there are only about 20 of them :) )

Reply:
oh, I see the issue
(btw, if unsure if style.css is used anywhere, then keep this change in style.css)
Will also need to make these changes in all the other themes (note this code block is in all of them). If this is overwhelming, I'm happy to do this since it gives me a chance to try cool things with phpstorm :)